<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qguide</class>
 <widget class="QWidget" name="Qguide">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>961</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="right" native="true">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>401</width>
     <height>911</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>到达点</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>可在图上点击点,选点</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>起点</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="start_edit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="end_edit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="distance">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>最短路长：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>740</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>下一步</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="SetofEnd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>361</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="accuary">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>73</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>最优路径</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="SA">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>73</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>较优路径</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="startbutton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>起点</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="endbutton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>到达点</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="callButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>700</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>计算最短路</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>780</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QLabel" name="dis">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>610</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="picture">
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/data/map.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="draw">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>960</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>picture</zorder>
  <zorder>right</zorder>
  <zorder>draw</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
